src/discussions/comments/reply/Reply.jsx

```diff
@@ -0,0 +1,56 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import * as timeago from 'timeago.js';
+import CommentIcons from '../comment-icons/CommentIcons';
+import { commentShape } from '../comment/Comment';
+import { selectCommentResponses } from '../data/selectors';
+import { fetchCommentResponses } from '../data/thunks';
+import messages from '../messages';
+
+function Reply({
+  reply,
+  intl,
+}) {
+  const dispatch = useDispatch();
+  const hasParent = !!reply.parent_id;
+  const inlineReplies = useSelector(selectCommentResponses(reply.id));
+  useEffect(() => {
+    // If the comment has a parent comment, it won't have any children, so don't fetch them.
+    if (!hasParent) {
```

Comment:
Seems more intuitive to name this `hasChildren`:
```js
const hasChildren = !reply.parent_id;
// ...
if (hasChildren) {
  dispatch(fetchCommentResponses(reply.id));
}
```

Reply:
I think that's a lot cleaner. Will do.